### ISSUE TITLE
GameMode is being invoked twice on some games

### ISSUE THREAD

**Issue by thomas-mc-work** [FeralInteractive/gamemode#319]
**Describe the bug**

I'm using it with a remake of [Need For Speed II SE](https://github.com/zaps166/NFSIISE). The game is starting fine with `gamemoderun` prepended. But the gamemode is being started twice, which is a problem for one if my actions (flipping [redshift](https://github.com/jonls/redshift/): double flip turns it off and then on again).

**To Reproduce**

Steps used to reproduce the behavior:
1. Launch gamemod with `/usr/bin/gamemoded`
2. Run game `gamemoderun ./nfs2se`

**Expected behavior**

It would be nice to have the [start] commands in the `[custom]` section of the config file being invoked only once per start process instead of twice.

**System Info (please complete the following information):**
 - OS and version: Ubuntu 20.04
 - GameMode Version:1.6.1 from Debian Sid

**Additional context**

I'm not sure whether this is caused by GameMode or the startup behavior of the game itself.

**Comment by thomas-mc-work** [comment]
Just recognized that all my games are affected. This has occurred after my upgrade from version `1.5.1-0ubuntu3.1`

**Comment by Figuera** [comment]
It happens to me too, I think it is a consequence of having two level of gamemode configuration. One at `/usr/share/gamemode/gamemode.ini´ and the other at `~/.config/gamemode.ini`. I solved it by removing one of them...

**Comment by stephanlachnit** [comment]
You should not launch `gamemoded` by hand, as gamemoderun kinda starts it itself. This might be the problem, since you (maybe) now have two daemons listen. Does this also happen, if you don't launch `gamemoded` before?

@Figuera This should not be a problem. The ini files should be merged. The only problem I see is if you have the custom section twice.

**Comment by Figuera** [comment]
@stephanlachnit 

I understand, but doing that did indeed solved the problem I was having.

That was some time ago so I wonder if it is still a problem.
